helm/ds/values.yaml

```diff
@@ -35,10 +35,10 @@ idmRepo:
   enabled: true
 
 image:
-  repository: forgerock-docker-public.bintray.io/forgerock/ds-paas
-  pullPolicy: IfNotPresent
-  #pullPolicy: Always
-  tag: 7.0.0-M134.2
+  repository: forgerock-docker-public.bintray.io/forgerock/ds
```

Comment:
why is this changing?  Is /ds for snapshots and /ds-paas for milestones & release builds?

Reply:
ds-paas is gone
it is just ds now

Reply:
:thumbsup: